openmc/deplete/results_list.py

```diff
@@ -57,15 +57,24 @@ def get_atoms(self, mat, nuc):
             Material name to evaluate
         nuc : str
             Nuclide name to evaluate
+        nuc_units : {"atoms", "atoms/b/cm", "atoms/cm^3"}, optional
```

Comment:
To stay consistent with the units we use in material.py for the `Material` class, I would suggest
```suggestion
        nuc_units : {"atoms", "atom/b-cm", "atom/cm3"}, optional
```

Reply:
Ah, excellent point. I'll add those